kontrakter/dom/domsavgjoerelse/arbeidsversjon/eksempelfiler/domsavgjoerelse-eksempel-1.json

```diff
@@ -0,0 +1,254 @@
+                  ]
+                },
+                "bot": {
+                  "beloep": 24000,
+                  "proevetid": {
+                    "lengde": {
+                      "antallAar": 2
+                    }
+                  },
+                  "subsidaerFengseltraff": {
+                    "lengde": {
+                      "antallDager": 24
+                    }
+                  }
+                },
+                "fengsel": {
+                  "straff": {
+                    "lengde": {
+                      "antallAar": 2,
+                      "antallMaaneder": 4
+                    }
+                  },
+                  "betinget": {
+                    "lengde": {
+                      "antallMaaneder": 6
+                    }
+                  },
+                  "proevetid": {
+                    "lengde": {
+                      "antallAar": 2
+                    }
+                  },
+                  "fradrag": {
+                    "lengde": {
+                      "antallDager": 24
+                    }
+                  },
+                  "saervilkaar": [
+                    {
+                      "paragraf": "§35",
+                      "tekst": "Særvilkår om meldeplikt",
+                      "beskrivelse": "Masse Kiwi skal melde seg til politiet én gang i måneden"
+                    },
+                    {
+                      "paragraf": "§36",
+                      "tekst": "Andre særvilkår",
+                      "detaljer": "f. gjennomføre narkotikaprogram med domstolskontroll, jf. § 38,"
+                    }
+                  ]
+                }
+              }
+            ]
+          },
+          {
+            "personForetak": {
+              "personRef": "5d516a2c-3283-4636-961a-456e1f26cea4"
+            },
+            "resultat": [
+              {
+                "basissakId": "2",
+                "lovbudKombinert": {
+                  "lovbudKombinertId": "61AE16B2-E360-451E-82B5-2D2C3AD68FA3"
+                },
+                "grunnlagstekst": "Søndag 29. oktober 2017 kl. 15.53 i Sponvikveien 7 i Halden, som deltaker i en samhandlende gruppe, medvirket de til innførsel av 3 600 liter rødvin av typen Torrero, 11 700 liter øl av typen Carlsberg og Tuborg, samt 1441,44 liter Coca Cola. B foresto det hele ved kontakt med N2 som deretter leide inn N3 som sjåfør. A lot varene betales med sitt bankkort kort forut for at varene ble innført til Norge. Ved vurderingen av om tollovertredelsen er grov, er det særlig lagt vekt på beslagets størrelse og beregnet avgiftsunndragelse",
+                "resultatType": "FRIKJENT",
+                "straffesaksnummer": "131221"
+              }
+            ],
+            "reaksjoner": []
+          }
+        ]
+      },
+      "erklaering": {
+        "erklaeringPaatale": {
+          "meddeltIRettsmoete": true,
+          "erklaert": "VEDTATT"
+        },
+        "erklaeringSiktede": {
```

Comment:
Dersom det er flere siktede/tiltalte, kan de anke hver for seg?

Reply:
Godt poeng. Er jo bare èn siktet i tilståelse-saker, men denne kontrakten skal håndtere alle doms-avgjørelser. Kanskje vi bør legge erklæringen under hver person i avgjørelsen. Jeg må tenke litt på den...

Reply:
Erklæring flyttet slik at det nå ligger under hver enkelt person